src/encoded/tests/test_file.py

```diff
@@ -4,14 +4,12 @@
 import tempfile
 
 from dcicutils.beanstalk_utils import source_beanstalk_env_vars
-from dcicutils.misc_utils import file_contents
-from dcicutils.qa_utils import MockBoto3, MockBotoS3Client
 from pyramid.httpexceptions import HTTPForbidden
 from unittest import mock
 from ..types.file import FileFastq, post_upload, external_creds
 
 
-pytestmark = [pytest.mark.setone, pytest.mark.working]
+pytestmark = [pytest.mark.setone, pytest.mark.working, pytest.mark.integrated]
```

Comment:
Marking integrated here means everything in this file is an integrated test. I don't think you want that. I'd remove this marker and leave the others on individual functions.

Reply:
Removed and added an inline comment. Will merge when CI completes.